src/units/pathfinder2remaster/player-core/ancestry/dwarf/heritage/heritage-remaster-dwarf-ancient-blooded

```diff
@@ -0,0 +1,23 @@
+unit: heritage/dwarf/remaster/ancient-blooded
+in: heritage/dwarf
+group: "_{Player Core}"
+name: "_{Ancient-Blooded Dwarf}"
+
+inc:
+  - set: heritage
+    value: "_{Ancient-Blooded Dwarf}"
+
+  - at: '@heritage'
+    replace:
+      - article:
+        title: "_{Ancient-Blooded Dwarf}"
+        contents:
+          - p: "_{Shrug off your enemies' magic. Call on ancient blood to gain a bonus to saving throws.}"
```

Comment:
Skip the flavour text, replace it with the rules change only (even if that's a dupliate of what's in the Saving Throws section)